lists/cl/cl-mp.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": {
+    "en": "Public Market Suppliers and Buyer Units, Chile",
+    "local": "Proveedor y Unidad Comprador Mercado Público, Chile"
+  },
+  "url": "https://www.mercadopublico.cl/Home",
+  "description": {
+    "en": "Mercado Público is an electronic platform managed by Chilecompra, part of the Ministry of Finance, which public organizations use to make purchases.\n\nOrganizations registered as buyers or suppliers on the Mercado Público platform are assigned an identifier in the system. This identifier is used in the data from the Mercado Público OCDS API.\n\nThe use of Mercado Público identifiers is considered a temporary measure. They should only be used when no other stable identifier for a government agency is available"
+  },
+  "coverage": [
+    "CL"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "government_agency"
+  ],
+  "sector": [],
+  "code": "CL-MP",
+  "confirmed": false,
```

Comment:
This should be 'true' or the list will not appear in the listings on the org-id.guide website

Reply:
Updated in https://github.com/org-id/register/pull/424/commits/ca9c0f0eec1e827ccb53925860ca46ecf386ac58. Should I just set it to `true` in future PRs even before they have been reviewed?

Reply:
@duncandewhurst - yes. I know that the schema description states

`Once this list has been reviewed by an administrator, this value can be set as true`

But the practicality is that the list will only go live if it has been reviewed and we risk valid lists not showing in the UI. 

Our main use for 

`"confirmed": false`

is for lists that have been imported automatically (from OpenCorporates mainly) but have not been worked up into a full org-id list yet.

Reply:
Understood :-)